pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -160,7 +160,8 @@ public enum ControllerGauge implements AbstractMetrics.Gauge {
 
   TABLE_DISABLED("tableDisabled", false),
 
-  TABLE_REBALANCE_IN_PROGRESS("tableRebalanceInProgress", false);
+  TABLE_REBALANCE_IN_PROGRESS("tableRebalanceInProgress", false),
+  TABLE_STORAGE_QUOTA_EXCEEDED("tableStorageQuotaExceeded", true);
```

Comment:
We should not need a metric saying quota is exceeded. Instead, let us have a metric that emits the percent of quota used by the table. Alerts can be set in specific destinations on the value of this metric. Some installations can choose to set the alert at 80%, others at 90%, etc. depending on the action they want to take.

The value can also be greater than 100%, btw.

Reply:
I have removed the `tableStorageQuotaExceeded` gauge. Will let the downstream system figure out the logic based on the `TABLE_QUOTA` and `TABLE_TOTAL_SIZE_ON_SERVER` gauges.